File: RepoTransBench/target_projects/Java/Go/googlearchive_android-BluetoothLeGatt/public_tests/sample_gatt_attributes_edge_public_test.go
package public_tests

import (
	"testing"

	"github.com/stretchr/testify/assert"
	"bluetoothlegatt/tests"
)

func TestSampleGattAttributesEdgePublic_LookupNullUuidReturnsAlternateDefault(t *testing.T) {
	result := tests.SampleGattAttributes.Lookup("", "alt-default")
	assert.Equal(t, "alt-default", result)
}

func TestSampleGattAttributesEdgePublic_LookupEmptyUuidReturnsAnotherDefault(t *testing.T) {
	result := tests.SampleGattAttributes.Lookup("", "no-value")
	assert.Equal(t, "no-value", result)
}

func TestSampleGattAttributesEdgePublic_LookupNullDefaultReturnsNullForAnotherUnknown(t *testing.T) {
	result := tests.SampleGattAttributes.Lookup("another-notfound-uuid", nil)
	assert.Equal(t, "", result)
}

File: RepoTransBench/target_projects/Java/C#/googlearchive_android-BluetoothLeGatt/public_tests/SampleGattAttributesEdgePublicTest.cs
using Xunit;
using BluetoothLeGatt;

namespace PublicTests
{
    public class SampleGattAttributesEdgePublicTest
    {
        [Fact]
        public void Lookup_NullUuid_ReturnsAlternateDefault()
        {
            Assert.Equal("alt-default", SampleGattAttributes.Lookup(null, "alt-default"));
        }

        [Fact]
        public void Lookup_EmptyUuid_ReturnsAnotherDefault()
        {
            Assert.Equal("no-value", SampleGattAttributes.Lookup("", "no-value"));
        }

        [Fact]
        public void Lookup_NullDefault_ReturnsNullForAnotherUnknown()
        {
            Assert.Null(SampleGattAttributes.Lookup("another-notfound-uuid", null));
        }
    }
}